src/Categories/Category/Monoidal/Instance/Endofunctors.agda

```diff
@@ -0,0 +1,131 @@
+{-# OPTIONS --without-K --safe #-}
+module Categories.Category.Monoidal.Instance.Endofunctors {o ℓ e} where
+
+-- Given a category C, the category of endofunctors on C is a monoidal category.
+
+open import Data.Empty.Polymorphic using (⊥; ⊥-elim)
+open import Data.Product using (_,_; uncurry′)
+open import Data.Sum using (_⊎_; inj₁; inj₂; [_,_]′)
+open import Data.Unit.Polymorphic using (⊤)
+open import Function using (case_of_; flip)
+open import Level using (Lift; lift; _⊔_)
+open import Relation.Binary.PropositionalEquality using (_≡_; refl)
+
+open import Categories.Morphism using (_≅_; Iso)
+open import Categories.Category.Cartesian using (Cartesian)
+open import Categories.Category.Cartesian.Monoidal using (module CartesianMonoidal)
+open import Categories.Category.Core using (Category)
+open import Categories.Category.Cocartesian using (Cocartesian)
+open import Categories.Category.Construction.Functors using (Functors)
+open import Categories.Functor using (Functor; _∘F_) renaming (id to idF)
+open import Categories.NaturalTransformation using (NaturalTransformation; _∘ₕ_)
+open import Categories.Category.Monoidal using (Monoidal; monoidalHelper)
+open import Categories.Category.Monoidal.Symmetric using (Symmetric; symmetricHelper)
+open import Categories.NaturalTransformation.NaturalIsomorphism using (niHelper)
+
+open import Categories.NaturalTransformation.NaturalIsomorphism
+  using (NaturalIsomorphism; unitorˡ; unitorʳ; associator)
+
+import Categories.Morphism.Reasoning as MR
+
+Endofunctors : Category o ℓ e → Category (o ⊔ ℓ ⊔ e) (o ⊔ ℓ ⊔ e) (o ⊔ e)
+Endofunctors C = Functors C C
+
+module _ where
+  -- Conversion function from natural isomorphisms to isomorphisms in the functor category.
+  ≃→≅ : ∀ {C D : Category o ℓ e}
+      → {F G : Functor C D}
+      → NaturalIsomorphism F G
+      → Categories.Morphism._≅_ (Functors C D) F G
+  ≃→≅ i = record
+    { from = i.F⇒G
+    ; to = i.F⇐G
+    ; iso = record { isoˡ = λ {x} → Iso.isoˡ (i.iso x) ; isoʳ = λ {x} → Iso.isoʳ (i.iso x) }
+    } where module i = NaturalIsomorphism i
```

Comment:
I think this is the same as the conversions in [Categories.NaturalTransformation.NaturalIsomorphism.Functors](https://github.com/agda/agda-categories/blob/master/src/Categories/NaturalTransformation/NaturalIsomorphism/Functors.agda).